### PR TITLE
check for 127.0.1.1 in /etc/hosts

### PR DESCRIPTION
**Pull request by mmoll**
This was added to the installation instructions, but still people are showing up in IRC with this problem.

### CODE REVIEW

File: checks/debian_hosts.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+begin
```
Comment (ekohl) [inline checks/debian_hosts.rb]: Why this optional import?

Comment (mmoll) [inline checks/debian_hosts.rb]: Copied over from hostname.rb, I think that's because Ruby 1.8 needs that `require 'rubygems'` explicitly.

Comment (mmoll) [inline checks/debian_hosts.rb]: hrm, looking again, I don't use any gems in that file, so that can indeed go

**Comment by ekohl** [comment]
:+1:

**Comment by domcleal** [comment]
Thanks @mmoll!

**Comment by lzap** [comment]
Nice, I think this could be named differently as this is also useful for non-Debian hosts, but whatever :-)

**Pull request merged**